lib/Agrammon/DB/Dataset.pm6

```diff
@@ -329,16 +329,21 @@ class Agrammon::DB::Dataset does Agrammon::DB {
         }
     }
 
-    method delete-input($var-name) {
-        my $instance;
+    method delete-instance($variable-pattern, $instance) {
+        return unless $variable-pattern and $instance;
```

Comment:
This will return `Nil`, which will be mapped into a JSON `null`. Probably not so desirable?

Reply:
Added exception handler instead.